setup.py

```diff
@@ -143,4 +143,7 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     classifiers=CLASSIFIERS,
     install_requires=["boto>=2.31.0", "pytz>=2016.10", "future>=0.16.0"],
     include_package_data=True,
+    extras_require={
+        'bounce': ['requests', 'M2Crypto'],
```

Comment:
Should we require specific versions? Maybe the ones you tested with?

Reply:
I don't think we need to add specific versions right now. At least, I don't think we need any lower limit. I think basically any recent enough versions (within the past two years) of the two packages will work.

- If anybody has already been using the django-ses's bounce handling, then great, they must already have working versions installed, so not adding the `bounce` extra won't break anything;
- If somebody decides to start using django-ses's bounce handling today, then the latest versions available today on PyPi are also guaranteed to work.

However, if we specify something too strict, the dependency graph could become unresolvable because other parts of a user's project could somehow depend on `requests` and `M2Crypto` as well.

With regards to `requests`, it *may* be a good idea to specify and _upper_ limit `requests<3` because I know version 3 is being worked on and there may be breaking API changes.

Meanwhile, M2Crypto seems to move so slowly that it shouldn't be necessary to put an upper limit until we see an upcoming deprecation discussed in their changelog?

Reply:
Thanks for this. Alright, we can leave as is, and hope someone will file an issue if requests 3 breaks with this project.